src/reachy2_sdk/orbita/orbita.py

```diff
@@ -135,72 +124,22 @@ def temperatures(self) -> Dict[str, Any]:
         """Get temperatures of all the motors of the actuator"""
         return {motor_name: m.temperature for motor_name, m in self._motors.items()}
 
-    # def _set_motors_fields(self, field: str, value: float) -> None:
-    #     """Set the value of the register for all motors of the actuator.
-
-    #     It is used to set pid, speed_limit and torque_limit.
-    #     """
-    #     for m in self._motors.values():
-    #         m._tmp_fields[field] = value
-
-    #     self._update_loop(field)
-
-    def _setup_sync_loop(self) -> None:
-        """Set up the async synchronisation loop.
-
-        The setup is done separately, as the async Event should be created in the same EventLoop than it will be used.
-
-        The _need_sync Event is used to inform the robot that some data need to be pushed to the real robot.
-        The _register_needing_sync stores a list of the register that need to be synced.
-        """
-        self._need_sync = asyncio.Event()
-        self._loop = asyncio.get_running_loop()
-
-    def _update_loop(self, field: str) -> None:
-        """Update the registers that need to be synced.
-
-        Set a threading event to inform the stream command thread that some data need to be pushed
-        to the robot.
-        """
-
-        async def set_in_loop() -> None:
-            self._register_needing_sync.append(field)
-            self._need_sync.set()
-
-        fut = asyncio.run_coroutine_threadsafe(set_in_loop(), self._loop)
-        fut.result()
+    def _set_compliant(self, compliant: bool) -> None:
+        command = Orbita2dsCommand(
+            cmd=[
+                Orbita2dCommand(
+                    id=ComponentId(id=self._id),
+                    compliant=BoolValue(value=compliant),
+                )
+            ]
+        )
+        self._stub.SendCommand(command)
+
+    def _ask_for_new_goal_position(self, axis_name: str, goal_position: float) -> None:
```

Comment:
That was more confusing for me. Why not simply _set_waiting_goal_position? (or outgoing or whatever)
My point is that we directly know what this function does reading its signature. to me ask is like we're expecting an answer from something

Reply:
renamed